content/blog/autonaming-configuration/index.md

```diff
@@ -9,6 +9,7 @@ authors:
 tags:
     - features
     - releases
+canonical_url: "https://www.pulumi.com/docs/iac/concepts/resources/names/"
```

Comment:
If and only if canonical URLs can link to an anchor, otherwise disregard:

```suggestion
canonical_url: "https://www.pulumi.com/docs/iac/concepts/resources/names/#autonaming-configuration"
```